Sources/Filters/General/AppendPolyData/index.js

```diff
@@ -4,9 +4,9 @@ import vtkPoints        from 'vtk.js/Sources/Common/Core/Points';
 import vtkPolyData      from 'vtk.js/Sources/Common/DataModel/PolyData';
 import { VtkDataTypes } from 'vtk.js/Sources/Common/Core/DataArray/Constants';
 
-import Constants        from 'vtk.js/Sources/Filters/General/AppendPolyData/Constants';
+import Constants        from 'vtk.js/Sources/Filters/General/Constants';
 
-const { PointPrecision } = Constants;
+const { VtkPointPrecision } = Constants;
```

Comment:
Your constant should not start with `Vtk` [see](https://github.com/Kitware/vtk-js/blob/master/Sources/Common/DataModel/DataSet/Constants.js)

Reply:
The VtkDataTypes list the naming for the VTK Types that an vtkDataArray can contain, hence the VTK in it. But it you look in that same file, the other constants don't have Vtk as prefix.

Reply:
I see. Cool, I'll fix it when rebasing.